gyp/pylib/gyp/generator/ninja.py

```diff
@@ -2250,7 +2250,10 @@ def GenerateOutputForConfig(target_list, target_dicts, data, params,
     master_ninja.rule(
       'copy',
       description='COPY $in $out',
-      command='rm -rf $out && cp -af $in $out')
+      if sys.platform in ('os390'):
```

Comment:
This is a python syntax error:
```python
gyp info spawn args   '-Goutput_dir=.' ]
Traceback (most recent call last):
  File "c:\workspace\nodegyp-test-commit\nodes\win2012r2\gyp\gyp_main.py", line 16, in <module>
    sys.exit(gyp.script_main())
  File "c:\workspace\nodegyp-test-commit\nodes\win2012r2\gyp\pylib\gyp\__init__.py", line 545, in script_main
    return main(sys.argv[1:])
  File "c:\workspace\nodegyp-test-commit\nodes\win2012r2\gyp\pylib\gyp\__init__.py", line 538, in main
    return gyp_main(args)
  File "c:\workspace\nodegyp-test-commit\nodes\win2012r2\gyp\pylib\gyp\__init__.py", line 514, in gyp_main
    options.duplicate_basename_check)
  File "c:\workspace\nodegyp-test-commit\nodes\win2012r2\gyp\pylib\gyp\__init__.py", line 91, in Load
    generator = __import__(generator_name, globals(), locals(), generator_name)
  File "c:\workspace\nodegyp-test-commit\nodes\win2012r2\gyp\pylib\gyp\generator\msvs.py", line 15, in <module>
    import gyp.generator.ninja as ninja_generator
  File "c:\workspace\nodegyp-test-commit\nodes\win2012r2\gyp\pylib\gyp\generator\ninja.py", line 2253
    if sys.platform in ('os390'):
     ^
SyntaxError: invalid syntax
```
You could do the `if` before hand near L2167, or use the [python ternary](http://book.pythontips.com/en/latest/ternary_operators.html)